traefik/imageroot/actions/create-module/20certificate

```diff
@@ -31,7 +31,7 @@ set -e
 exec 1>&2
 
 NAME="selfsigned"
-FQDN=$(hostname -f)
+FQDN="ns8.local"
```

Comment:
The .local suffix is reserved for other purposes. What about appending .test to the FQDN?

```suggestion
FQDN="$(hostname -f).test"
```

Reply:
The browser will complain, but as long as HSTS is not enable on that domain, it should not be a real problem.

Reply:
:raised_hands:

Reply:
In the end, I pick the solution proposed by @DavidePrincipi :)